Doc/using/windows.rst

```diff
@@ -602,6 +602,50 @@ existed)::
 
     C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files\Python 3.9
 
+.. _win-utf8-mode:
+
+UTF-8 mode
+==========
+
+.. versionadded:: 3.7
+
+Windows still uses legacy encodings for the system encoding (the ANSI Code
+Page).  Python uses it for the default encoding of text files (e.g.
+:func:`locale.getpreferredencoding`).
+
+It may cause trouble because the UTF-8 is widely used on the internet
```

Comment:
In addition to Eryk's recommendation, I'd also advise using "This may cause issues" instead of "This may cause trouble", and removing the "the" before UTF-8:

```suggestion
This may cause issues because UTF-8 is widely used on the internet
```